jujugui/static/gui/src/app/components/added-services-list/test-added-services-list.js

```diff
@@ -308,26 +287,23 @@ describe('AddedServicesList', () => {
           services={services}/>, true);
 
     var instance = renderer.getMountedInstance();
-    // Call the fade Service method which would be passed down to the children.
-    instance.fadeService('trusty/wordpress');
-    assert.equal(serviceSet.callCount, 1);
-    assert.deepEqual(serviceSet.args[0], ['fade', true]);
+    var unfocusService = sinon.stub(instance, 'unfocusService');
+    instance.fadeService(id);
+    // Set the fade switch appropriately.
+    assert.equal(allServices[0].get('fade'), true);
+    // Unfocused the service.
+    assert.isTrue(unfocusService.calledWith(id));
+    unfocusService.restore();
```

Comment:
If it isn't stubbed (i.e., just use `sinon.spy()`) then a bunch of stuff happens in order to unfocused the service, which would in turn need to be mocked/stubbed in order for the test to pass. Best to stub. I removed the restore as I don't think it's actually a problem.